src/ShallowWrapper.js

```diff
@@ -3,6 +3,7 @@ import flatten from 'lodash/flatten';
 import unique from 'lodash/uniq';
 import compact from 'lodash/compact';
 import cheerio from 'cheerio';
+import merge from 'lodash/merge';
```

Comment:
oh, okay. i got it. I'll fix this and then send PR 😄 

thanks for your feedbacks 👍